store/tikv/client_batch.go

```diff
@@ -341,7 +374,6 @@ func (a *batchConn) batchSendLoop(cfg config.TiKVClient) {
 		a.fetchAllPendingRequests(int(cfg.MaxBatchSize), &entries, &requests)
 
 		if len(entries) < int(cfg.MaxBatchSize) && cfg.MaxBatchWaitTime > 0 {
-			tikvTransportLayerLoad := atomic.LoadUint64(batchCommandsClient.tikvTransportLayerLoad)
```

Comment:
You can get `tikvTransportLayerLoad` from `batchConn` instead of `batchClmmandsClient`.

Reply:
Done